schemas/overlay-terraform-role.yaml

```diff
@@ -23,6 +23,11 @@ actions:
     update:
       x-speakeasy-entity-operation: Role#update
 
+  - target: $.paths["/organizations/{organization}/databases/{database}/branches/{branch}/roles"].post.requestBody.content["application/json"].schema.properties.ttl
```

Comment:
This overlay *shouldn't* be necessary as we're correctly annotating the type internally in our API, but for whatever reason that's not working as I'd expect.